res/css/photon-components.css

```diff
@@ -23,7 +23,7 @@
 
 /* Using -moz-focusring to target Firefox only, as a result of the style above */
 .photon-button:-moz-focusring {
-  outline: 1px dotted black;
+  box-shadow: 0 0 0 1px #0a84ff inset, 0 0 0 1px #0a84ff, 0 0 0 4px rgba(10, 132, 255, 0.3);
 
   /* This should be eventually the default behavior.
    * See https://bugzilla.mozilla.org/show_bug.cgi?id=315209 */
```

Comment:
I think we don't need the outline radius below... because we don't use an outline anymore :)